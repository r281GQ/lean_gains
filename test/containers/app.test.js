import React from 'react';
import { shallow } from 'enzyme';
import App from './../../src/containers/app';

describe('App test', () => {
  it('should render', () => {
    const component = shallow(
      <App/>
    );
    expect(component).toHaveLength(1);
    expect(component).toMatchSnapshot();
  });
});
