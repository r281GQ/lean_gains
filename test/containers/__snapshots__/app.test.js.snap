// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test should render 1`] = `
<div>
  <withRouter(Connect(HeaderContainer)) />
  <Route
    component={[Function]}
    path="/app"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/login"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/signup"
  />
  <Redirect
    push={false}
    to="/app"
  />
  <FooterContainer />
</div>
`;
